libs/auth/src/reducers/auth-initial-state.ts

```diff
@@ -0,0 +1,8 @@
+import { DaffAuthReducerState } from './auth-reducer-state.interface';
+import { DaffAuthToken } from '../models/auth-token';
+
+export const daffAuthInitialState: DaffAuthReducerState<any> = Object.freeze({
```

Comment:
Why freeze the Object this is inconsistent with other packages?

Reply:
Unless we specifically want to allow the initial state to be modified it seems safest to freeze it.

Reply:
Removed `Object.freeze` from the initial states